stdlib/public/core/Array.swift

```diff
@@ -1735,10 +1746,12 @@ extension Array {
     @lifetime(&self)
     @_alwaysEmitIntoClient
     mutating get {
+#if INTERNAL_CHECKS_ENABLED && COW_CHECKS_ENABLED
```

Comment:
This doesn't make sense. If checks are enabled then you call the unchecked version?
This completely disables the checks in all cases.
Is this intention?

Reply:
Yes. This is disabling runtime verification for mutableSpan property completely. I will remove `#else` parts to remove the confusion.